src/Installer/PackagesInstaller.php

```diff
@@ -60,12 +50,10 @@ class PackagesInstaller implements InstallerInterface
                 'version' => '~0.30',
                 'updateDependencies' => true,
             ],
-        ],
-        'laravel' => [
-            'elgentos/laravel-coding-standard' => [
-                'version' => '^1.0.0',
+            'magento/magento-coding-standard' => [
+                'version' => '@stable',
```

Comment:
Should we make some mention that it is advised to override this to your preference in your project?
Maybe instead of @stable we should do >123 or so, think there's some obvious candidate for magento 2.4.4 / the lowest supported php version

No blocker thouhg

Reply:
I'm not sure what the benefit would be from using a > x constraint over just @stable. The stable tag clearly states intent. Only downside is we would override upstream minimum-stability tags since we don't just install 13-alpha or rc1 etc. upstream as soon as it's available.

Additionally, the testing suite is not a Magento tester. Sure magento 2.4.4 might have a minimal version but I don't really want to keep track of minimal php / phpunit versions for pimcore and drupal (and other frameworks/systems in the future since we want to work system agnostically).

I will add the hint about updating phpunit upstream and fixing the version constraint to the migration guide.